var/spack/repos/builtin/packages/intel-mpi/package.py

```diff
@@ -54,6 +54,13 @@ def mpi_libs(self):
             libraries, root=self.prefix.lib64, shared=True, recurse=True
         )
 
+    @property
+    def mpi_headers(self):
```

Comment:
supposedly that's how one overrides `spec['mpi'].headers ` which returns `HeaderList` object which by default looks for '*' headers and is recursive. Looking for `mpi.h` in `include64` looks ok to me. Hopefully `include64` always exists.